src/initFirebaseAdminSDK.js

```diff
@@ -3,17 +3,20 @@ import { getConfig } from 'src/config'
 
 const initFirebaseAdminSDK = () => {
   if (!admin.apps.length) {
-    const { firebaseAdminInitConfig } = getConfig()
-    if (!firebaseAdminInitConfig) {
+    const { firebaseAdminInitConfig, firebaseAdminDefaultCredential } =
+      getConfig()
+    if (!firebaseAdminInitConfig && !firebaseAdminDefaultCredential) {
       throw new Error(
         'If not initializing the Firebase admin SDK elsewhere, you must provide "firebaseAdminInitConfig" to next-firebase-auth.'
```

Comment:
Update to: "Missing Firebase admin credentials in next-firebase-auth. Set "firebaseAdminInitConfig", use default credentials, or initialize Firebase admin yourself."

Reply:
Updated